spec/Section 5 -- Validation.md

```diff
@@ -198,6 +198,67 @@ query getName {
 }
 ```
 
+### Subscription Operation Definitions
+
+#### Single root field
+
+**Formal Specification**
+
+  * For each subscription operation definition {subscription} in the document
+  * Let {rootFields} be the top level selection set on {subscription}.
+    * {rootFields} must be a set of one.
+
+**Explanatory Text**
+
+Subscription operations must have exactly one root field.
```

Comment:
I listed most of the points I found as I was implement it in sangria here:

https://github.com/facebook/graphql/issues/282#issuecomment-284584160

I think that it boils down to points `3.i`, `5` and `6`. My take on it is here: https://github.com/facebook/graphql/issues/282#issuecomment-284586368

IMHO, if we need to make a trade off, I would rather disallow not-null root subscription field types than allow only a single subscription field in a query.